packages/connect-query/src/connect-query-key.test.ts

```diff
@@ -90,17 +93,11 @@ describe("createConnectQueryKey", () => {
     const key = createConnectQueryKey({
       schema: ElizaService.method.say,
       input: skipToken,
+      cardinality: "finite",
     });
     expect(key[1].input).toBe("skipped");
   });
 
-  it("sets cardinality finite by default", () => {
-    const key = createConnectQueryKey({
-      schema: ElizaService.method.say,
-    });
-    expect(key[1].cardinality).toBe("finite");
-  });
```

Comment:
Maybe it would be helpful to cover that omitting cardinality in the params doesn't set it in the key?

Reply:
I think this is actually covered in `allows to set cardinality: undefined`. Although the way that I built the types, you actually can't omit cardinality, it is mandatory (even if it's set to undefined).